src/numpy.jl

```diff
@@ -175,14 +175,20 @@ const npy_typestrs = Dict( "b1"=>Bool,
 #########################################################################
 # no-copy conversion of Julia arrays to NumPy arrays.
 
-function PyObject{T<:NPY_TYPES}(a::StridedArray{T})
+# In some cases, an array is already in row-major order (e.g., RGB
```

Comment:
I would say rather than "Julia arrays are in column-major order, but in some cases it is useful to pass them to Python as row-major arrays simply by reversing the dimensions. For example, although NumPy works with both row-major and column-major data, some Python libraries like OpenCV seem to require row-major data (the default in NumPy).  In such cases, set `revdims=true`."

And then add a similar comment to the README.md file.